docs/getting_started/tutorial.md

```diff
@@ -146,6 +146,19 @@ After the server starts, go to <http://127.0.0.1:8000> to see Wagtail’s welcom
 ```{note}
 This tutorial uses `http://127.0.0.1:8000` as the URL for your development server but depending on your setup, this could be a different IP address or port. Please read the console output of `manage.py runserver` to determine the correct URL for your local site.
 ```
+### Running and Writing Tests
+
+Wagtail now includes basic tests in the `home/tests.py` file when you create a new project using `wagtail start`. These tests check:
```

Comment:
```suggestion
When you create a new project using `wagtail start`, there will be a set of basic tests included in the `home/tests.py` file.

These tests check:
```

Just a reword, we do not need to say 'now includes' as these docs will be read when the updates already exist. Also, we try to avoid repeating 'Wagtail' as it's the docs you are already looking at.